product_docs/docs/pem/9/monitoring_performance/probes.mdx

```diff
@@ -256,6 +259,52 @@ If you specify a parent node in the **Copy Probe Configuration** tree, PEM copie
 
 After you select the target object or objects, select **Configure Probes** to copy the probe definitions to the location selected in the dialog box.
 
+## Managing probes using pemworker
+
+You can enable and disable a probe using the pemworker utility from version 9.6 and later.
```

Comment:
Add the option here, the statment could be like this:-
You can enable and disable a probe using the pemworker utility from version 9.6 and later either by --probe-display-name, --probe-internal-name or by --probe-id